.github/workflows/deploy-ecs.yml

```diff
@@ -0,0 +1,39 @@
+name: Deploy to ECS
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        required: true
+        default: dev
+  push:
+    branches: ["main"]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    environment: ${{ inputs.environment || 'dev' }}
+    concurrency: ${{ inputs.environment || 'dev' }}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: mbta/actions/build-push-ecr@v2
+        id: build-push
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          docker-repo: ${{ secrets.DOCKER_REPO }}
+      - uses: mbta/actions/deploy-scheduled-ecs@v2
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_ARN }}
+          ecs-cluster: linux-staging
```

Comment:
Question: Is this script just for non-prod UGPS environments at the moment/will it be updated for prod in the future? If it's intended for all envs, I don't think we want to use the `linux-staging` cluster for UGPS prod

Reply:
I was thinking just non-prod, but I went ahead and added it: 75248c7